backend/src/main/java/cz/syntaxbro/erpsystem/constants/Roles.java

```diff
@@ -0,0 +1,7 @@
+package cz.syntaxbro.erpsystem.constants;
+
+public class Roles {
```

Comment:
could be replaced by enum
```suggestion
public enum Roles {
    ADMIN("ROLE_ADMIN", 0),
    MANAGER("ROLE_MANAGER", 1),
    USER("ROLE_USER", 2);

    private final String name;
    private final int value;

    Roles(String name, int value) {
        this.name = name;
        this.value = value;
    }

    public String getName() {
        return name;
    }

    public int getValue() {
        return value;
    }
}
```

Reply:
I agree to use enum here.